utils/wasm-gen/src/config/module.rs

```diff
@@ -115,6 +119,12 @@ impl From<(SelectableParams, ArbitraryParams)> for WasmModuleConfig {
             max_memory_pages,
         } = arbitrary_params;
 
+        let allowed_instructions: Vec<_> = allowed_instructions
+            .into_iter()
+            .filter(|ix| !forbidden_instructions.contains(ix))
+            .collect();
+        let allowed_instructions = InstructionKinds::new(&allowed_instructions);
```

Comment:
I think `allowed_instructions` must go to `SelectableParams` too.

Reply:
Hm, but will we use it when setting up fuzzer? For now It's used only for tests

Reply:
You can have default implementation of it. I suggest moving it, because it seems wrong to define instructions in one config and then invalidate them in the other.